docs/rules/extensions.md

```diff
@@ -170,3 +170,6 @@ import foo from '@/foo';
 ## When Not To Use It
 
 If you are not concerned about a consistent usage of file extension.
+
+In the future, when this rule supports native node ESM resolution (and the plugin is configured to use native, instead of transpiled, ESM, a config option that is not yet available), setting this to `always` will have no effect.
```

Comment:
On second thought, this sentence clause has quite a few commas. What do you think of:

```suggestion
In the future, when this rule supports native node ESM resolution, and the plugin is configured to use native rather than transpiled ESM (a config option that is not yet available) - setting this to `always` will have no effect.
```

Reply:
sure, that's fine